python/pyspark/pandas/tests/test_dataframe_spark_io.py

```diff
@@ -99,7 +99,7 @@ def test_parquet_read_with_pandas_metadata(self):
             expected3 = expected2.set_index("index", append=True)
             # There is a bug in `to_parquet` from pandas 1.5.0 when writing MultiIndex.
             # See https://github.com/pandas-dev/pandas/issues/48848 for the reported issue.
-            if LooseVersion(pd.__version__) == LooseVersion("1.5.0"):
+            if LooseVersion(pd.__version__) in (LooseVersion("1.5.0"), LooseVersion("1.5.1")):
```

Comment:
If they look not going to fix this soon, let's just do the comparison with `< LooseVersion("1.5.0")`

Reply:
I'm not sure they it's fixed soon but seems trying to fix it ASAP, so I just wanna check explicitly for every version for now

Reply:
If it wouldn't fixed in 1.5.2, let me check is as `< LooseVersion("1.5.0")` then ?

Reply:
Let's just fix it `< LooseVersion("1.5.0")` here.

Reply:
Sure! Just addressed it